[DOCS] Wildcards and "matches" operator in Event Filters (#1823)
elastic/security-docs: docs/management/admin/event-filters.asciidoc

[chapter, role="xpack"]
= Event filters

> Event filters allow you to filter endpoint events that you do not need or want stored in {es} -- for example high-volume events. By creating event filters, you can optimize your storage in {es}. All endpoint events have the `endpoint.events.network` field.

Author: ```suggestion
Event filters allow you to filter endpoint events that you do not need or want stored in {es} -- for example, high-volume events. By creating event filters, you can optimize your storage in {es}. All endpoint events have the `endpoint.events.network` field.
```

Create event filters from the Hosts page or the Event filters page.

. Do one of the following:
+
--
* To create an event filter from the Hosts page:
.. Go to *Explore* -> *Hosts*.
.. Select the *Events* tab to view the Events table.
+

> .. Find the event to filter, click the More actions menu (*...*), then select *Add Endpoint event filter*.

Reviewer: ```suggestion
.. Find the event to filter, click the *More actions* menu (*...*), then select *Add Endpoint event filter*.
```

Reviewer: Also, it looks like we refer to this as a button in the "Manage detection alerts" topic.

Author: Good catch! @nastasha-solomon Since you're currently editing that topic for https://github.com/elastic/security-docs/pull/1836, could you revise that instance of "button" into "menu"?

> .. Find the event to filter, click the *More actions* button (*...*), then click *Add Endpoint event filter*.

Reviewer 2: ```suggestion
.. Find the event to filter, click the More actions menu (*...*), then select *Add Endpoint event filter*.
```

+
TIP: Since you can only create filters for endpoint events, be sure to filter the Events table to display events generated by the {elastic-endpoint}. +
In the KQL search bar, enter the following query: `event.dataset : endpoint.events.network`.

> * To create an event filter via the Event filters page:

Reviewer 2: ```suggestion
* To create an event filter from the Event filters page:
```

.. Go to *Manage* -> *Event filters*.
.. Click *Add Event Filter*. The *Add event filter* flyout opens.
--
+
[role="screenshot"]
image::images/event-filter.png[]
. Fill in these fields in the **Details** section:
  .. `Name your event filter`: Enter a name for the event filter.
  .. `Description`: Enter a filter description (optional).
. In the **Conditions** section, depending which page you're using to create the filter, either modify the pre-populated conditions or add new conditions to define how {elastic-sec} will filter events. Use these settings:
  .. `Select operating system`: Select the appropriate operating system.
  .. `Field`: Select a field to identify the event being filtered.
  .. `Operator`: Select an operator to define the condition. Available options are:
    * `is`
    * `is not`
    * `is one of`
    * `is not one of`

>     * `matches`: This is only available for the field `file.path.text`, and it allows you to use wildcards in `Value`, such as `C:\path\*\app.exe`.  Available wildcards are `?` (match one character) and `*` (match zero or more characters).

Reviewer 2: ```suggestion
    * `matches`: This is only available for the `file.path.text` field, and allows you to use wildcards in `Value`, such as `C:\path\*\app.exe`.  Available wildcards are `?` (match one character) and `*` (match zero or more characters).
```
